cgi-bin/DW/Controller/Support/History.pm

```diff
@@ -107,7 +107,7 @@ sub history_handler {
                 $reqs{$row->[0]} = {
                     spid => $row->[0],
                     winner => $points{$row->[0]}->[0],
-                    points => $points{$row->[0]}->[1]+0,
+                    points => $points{$row->[0]}->[1] || 0,
```

Comment:
The point of `+0` is to coerce to an integer, so your change here makes it so that a non-integer field doesn't get coerced like it was. So, really, this should be,

```
                    points => ($points{$row->[0]}->[1] // 0)+0,
```